core/php8.0Action/Dockerfile

```diff
@@ -17,91 +17,46 @@
 
 # build go proxy from source
 ARG GO_PROXY_BASE_IMAGE=golang:1.18
-FROM $GO_PROXY_BASE_IMAGE AS builder_source
+FROM $GO_PROXY_BASE_IMAGE AS builder
```

Comment:
This pattern is used in all the dockerfiles - what are we losing?

Reply:
Which pattern?

Reply:
Install from source vs release - should be in all the go proxy derived dockerfiles.

Reply:
We don't need the release based pattern, since we're not using a released binary. We always compile from code. As such, I don't think we're loosing anything. Node 18 follows this "new way" as well.